src/qpmodel.jl

```diff
@@ -1,31 +1,43 @@
+# override some NLPModels functions
+import NLPModels.jac_structure!, NLPModels.hess_structure!, NLPModels.jac_coord!, NLPModels.hess_coord!
+export jac_structure!, hess_structure!, jac_coord!, hess_coord!
```

Comment:
I think it's better to not import these functions and use `function NLPModels.foo` directly on the definition.